src/main/java/core/basesyntax/service/impl/WriteCsvService.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Writer;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriteCsvService implements Writer {
+    private static final String REPORT_PATH = "src/main/resources";
+    private static final String FILE_NAME = "Log.CSV";
```

Comment:
Not sure you need to split it into two variables. Furthermore, if someone needs to write to another path, should he go right here and change the path or it would be better to create a constant in Main?
Anyway, it's not "Log", but a report.